client/src/app/page.tsx

```diff
@@ -1,43 +1,68 @@
-import { Typography } from '@mui/material';
+import React from 'react';
+import Link from 'next/link';
+import Button from '../components/button';
+import Typography from '@mui/material/Typography';
+import { Box, Container, colors } from '@mui/material';
 
 import styles from './styles/home.module.scss';
-import Link from 'next/link';
-import Button from 'components/button';
 
 function Home() {
   return (
-    <main className={styles['home__main']}>
-      <div id="home-hero" className={styles['hero__box']}>
-        <div className={styles['hero__content-box']}>
-          <Typography id="hero-display" variant="display">
-            A Complete Map Visuals Studio
+    <main className="flex min-h-screen flex-col items-center justify-center p-24">
+      <Box
+        sx={{
+          height: '80px', // Set the height to 80px
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center', // Center the content horizontally
+        }}
+      >
+        {/* TODO: Add NavBar here */}
+      </Box>
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
+          width: '1513px',
+          height: '808px',
+          borderRadius: 32,
+          margin: '50px 191px 86px 50px',
+          bgcolor: 'primary.container'
+        }}
+      >
+        <Container sx={{
+          marginLeft: '50px',
+          marginBottom: '30px'
+        }}>
+          <Typography variant="h1">
+            A Complete <br/>
+            Map Visuals Studio.
           </Typography>
-        </div>
-        <div className={styles['hero__content-box']}>
-          <Typography id="hero-body" variant="body">
-            Five Essential Templates.
-            A Plethora of Editing Tools.
+          <Typography variant='body1' sx={{marginTop: '10px'}}>
+            Five Essential Templates.<br/>
+            A Plethora of Editing Tools.<br/>
             Unique data-driven approach.
           </Typography>
-        </div>
-        <div 
-          className={`
-            ${styles['hero__content-box']} 
-            ${styles['hero__content-box--buttons']}`
-          }
-        >
-          <Link id="hero-discover" href="/discover">
```

Comment:
Removing the Ids and changing the hrefs without changing the tests would break them.